App/Seller/Sources/Seller/UI/MessagesView/FilterSearchView.swift

```diff
@@ -0,0 +1,136 @@
+// system
+import SwiftUI
+
+struct FilterSearchView: View {
+    
+    @ObservedObject var viewModel: MyMessagesViewModel
+    private let cornerRadiusSize = 10.0
+    private let lightGrayBackground = Color(red: 0.95, green: 0.95, blue: 0.96)
```

Comment:
Maybe it's a good moment to move this to some sort of constants? That will be a good start for a style guide later :)

Reply:
I will create a seperate task, because it will be useful to create Theme / Colors / etc style constants file already